assets/react/components/show_post.js

```diff
@@ -14,6 +14,10 @@ export default class ShowPost extends Component {
 	show_user(id){
 		this.context.router.transitionTo('/profile/' + id);
 	}
+	formatPostDate(date){
```

Comment:
we should create helper file. I also created the same function in profile page. Create one i will also refer to that.